module/data/item/equipment.mjs

```diff
@@ -38,21 +41,22 @@ export default class EquipmentData extends ItemDataModel.mixin(
   static defineSchema() {
     return this.mergeSchema(super.defineSchema(), {
       type: new ItemTypeField({value: "light", subtype: false}, {label: "DND5E.ItemEquipmentType"}),
-      armor: new foundry.data.fields.SchemaField({
-        value: new foundry.data.fields.NumberField({required: true, integer: true, min: 0, label: "DND5E.ArmorClass"}),
-        dex: new foundry.data.fields.NumberField({required: true, integer: true, label: "DND5E.ItemEquipmentDexMod"})
+      armor: new SchemaField({
+        value: new NumberField({required: true, integer: true, min: 0, label: "DND5E.ArmorClass"}),
+        magicalBonus: new NumberField({min: 0, integer: true, label: "DND5E.MagicalBonus"}),
```

Comment:
Are there no items that grant a bonus equal to some roll data like your Charisma modifier? I feel like there must be. In which case it would need to preferably be a FormulaField.

Reply:
I have never seen anything that adds a variable bonus equally to both attack and damage.

Reply:
This is on armor, for AC bonus. :)